Formula/propka.rb

```diff
@@ -0,0 +1,23 @@
+class Propka < Formula
+  # cite Sondergaard_2011: "https://doi.org/10.1021/ct200133y"
+  # cite Olsson_2011: "https://doi.org/10.1021/ct100578z"
+  desc "Predict pKa values of ionizable groups in protein-ligand complexes"
+  homepage "http://propka.org/"
+  url "https://github.com/jensengroup/propka-3.1/archive/91925bb92092d3b180a9c086e81822a3fdec6939.tar.gz"
```

Comment:
Please open an issue upstream and ask the upstream authors to tag a stable release.

Reply:
Since there are no stable releases in the original GitHub website, I have modified the url:
>  url "https://files.pythonhosted.org/packages/a3/7b/e935ce37cbf9ac8fe23ec05f8072fb3a63582b20fb01ef0c24502b5d10ba/PROPKA-3.1.0.tar.gz"
>  sha256 "ada690067c5ad9ca6be412491380929e5275fa41166aa1fb2c820fcc2dfdb619"

This is a tgz file uploaded by PyPI project. Is it OK?

Reply:
Please open an issue upstream and ask the upstream authors to tag a stable release.
Please post a link to that issue here.

> This is a tgz file uploaded by PyPI project ( https://pypi.org/project/PROPKA/3.1.0/#files ). Is it OK?

Yes, that'd be okay, but I'd prefer a tagged stable release on GitHub.

Reply:
Thank you for your help. I have opened an issue on https://github.com/jensengroup/propka-3.1/issues/21 .